viewing_party/party.py

```diff
@@ -0,0 +1,214 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        # Return a dictionary that has three key-value pairs
+        # The three keys are title, genre, and rating
+        movie_dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    add_watchlist = user_data["watchlist"]
+    add_watchlist.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
```

Comment:
Great job 👍  Creating a separate list to remove items is a good way to avoid the common problem of modifying a list as you're iterating through it.